CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -349,7 +349,7 @@ CF_EXPORT void * __CFConstantStringClassReferencePtr;
 
 #if DEPLOYMENT_RUNTIME_SWIFT
 
-CF_EXPORT void *__CFConstantStringClassReference[];
+CF_EXPORT void *__CFConstantStringClassReference[] asm("_T010Foundation19_NSCFConstantStringCN");
```

Comment:
Yes this mangled name seems to be due to the metadata changes in march.
For Xcode, it looks like the change in https://github.com/apple/swift-corelibs-foundation/pull/1006 is required